client/site-profiler/components/get-report-form/index.tsx

```diff
@@ -109,13 +109,13 @@ export function GetReportForm( {
 		>
 			<div className="get-report-form__container">
 				<div className="get-report-form__title">
-					<span className="title">{ translate( 'Get full report' ) }</span>
+					<span className="title">{ translate( 'Full site report' ) }</span>
 				</div>
 				<div className="get-report-form__body">
 					<div className="get-report-form__header">
 						<span className="description">
 							{ translate(
-								'Enter your details below to receive the full report with detailed insights and recommendations for your site.'
+								'To access a detailed report complete with insights and tailored recommendations for improving your site‘s performance, please provide your details below. We‘ll send you an exclusive link to view the full analysis, helping you make informed decisions for optimizing your site.'
```

Comment:
```suggestion
								'To access a detailed report complete with insights and tailored recommendations for improving your site\'s performance, please provide your details below. We\'ll send you an exclusive link to view the full analysis, helping you make informed decisions for optimizing your site.'
```

The quote imported from Figma is a different character, not sure if that might become a problem in the future but I had replaced here.

Reply:
I committed this change, but prettier throws an error with this version, and auto-saves it back to be non-escaped.

Reply:
It actually changes to a string wrapped in double quotes which doesn't need escape, so problem solved anyway :D